arangod/RocksDBEngine/RocksDBIterators.cpp

```diff
@@ -193,26 +192,53 @@ RocksDBAnyIndexIterator::RocksDBAnyIndexIterator(
   TRI_ASSERT(_iterator);
 
   _total = col->numberDocuments(trx);
-  uint64_t off = RandomGenerator::interval(_total - 1);
+  _forward = RandomGenerator::interval(uint16_t(2)) ? true : false; //prefer forward
```

Comment:
This strongly prefers forward iteration, as the generated values will mean the following:
- 0: backward
- 1: forward
- 2: forward

Reply:
ok i'll make it totally fair again